includes/class-sensei-analysis.php

```diff
@@ -81,6 +84,74 @@ public function __get( $key ) {
 		}
 	}
 
+	/**
+	 * Add custom navigation to the admin pages.
+	 *
+	 * @since 4.2.0
+	 * @access private
+	 */
+	public function add_custom_navigation() {
+		// phpcs:ignore WordPress.Security.NonceVerification -- No action, nonce is not required.
+		$is_reports_page = isset( $_GET['page'] ) && ( self::PAGE_SLUG === $_GET['page'] );
+
+		if ( ! $is_reports_page ) {
+			return;
+		}
+
+		// phpcs:ignore WordPress.Security.NonceVerification -- No action, nonce is not required.
+		if ( isset( $_GET['course_id'] ) || isset( $_GET['lesson_id'] ) || isset( $_GET['user_id'] ) ) {
+			return;
+		}
+
+		$this->display_reports_navigation();
+	}
+
+	/**
+	 * Display the Reports navigation.
+	 */
+	private function display_reports_navigation() {
+		// phpcs:ignore
+		$type = isset( $_GET['view'] ) ? esc_html( $_GET['view'] ) : false;
+
+		$reports            = array(
+			'learners' => __( 'Students', 'sensei-lms' ),
```

Comment:
I wonder if we could name this key "students" as well since we're trying to move away from "learners"? It looks like this actually used to be named "users" though, which makes me wonder about the potential of breakage elsewhere in the codebase.